ledger/src/blockstore.rs

```diff
@@ -319,6 +319,11 @@ impl Blockstore {
             .expect("Database Error: Failed to get write batch");
         // delete range cf is not inclusive
         let to_slot = to_slot.checked_add(1).unwrap_or_else(|| std::u64::MAX);
+        // prune transaction statuses
+        if !self.is_transaction_status_empty() {
+            let _result =
+                self.prune_transaction_status_cf_by_slot(&mut write_batch, from_slot, to_slot);
+        }
         let columns_empty = self
```

Comment:
You can also use the `self.is_transaction_status_empty()` here to determine `columns_empty` right?

Reply:
actually I don't think you need to include to do that, we don't need to consider the transaction status column in this check because if the data column is empty, that should be sufficient to say this column is also empty

Reply:
Right :)